relay-server/Cargo.toml

```diff
@@ -57,15 +57,15 @@ relay-general = { path = "../relay-general" }
 relay-log = { path = "../relay-log" }
 relay-quotas = { path = "../relay-quotas" }
 relay-redis = { path = "../relay-redis" }
-reqwest = { version = "0.10.8", features = ["gzip", "stream"] }
+reqwest = { version = "0.11.0", features = ["gzip", "stream", "trust-dns"] }
```

Comment:
Let's upgrade to `0.11.1` to get some of the proxy fixes in.

Reply:
yup

Reply:
ran into native-tls issue here: https://github.com/seanmonstar/reqwest/issues/1181